server/src/main/java/CSC4410/CovidTracker/operation/query/CountyTableCreateQuery.java

```diff
@@ -0,0 +1,30 @@
+package CSC4410.CovidTracker.operation.query;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+/**
+ * A database query which creates the table for data by county, if it
+ * does not already exist.
+ */
+public class CountyTableCreateQuery extends Query {
+    @Override
+    protected String getStatementBody() {
+        return "CREATE TABLE IF NOT EXISTS county_data (" +
+                "county_fips INT PRIMARY KEY," +
```

Comment:
i think for the primary key is better to use long data type because we are only dealing with small values